scheduler-server/src/test/java/gov/nasa/jpl/aerie/scheduler/server/services/SchedulingIntegrationTests.java

```diff
@@ -697,6 +698,31 @@ export default (): Goal => {
     assertEquals(Map.of("peelDirection", SerializedValue.of("fromStem")), peelBanana.args());
   }
 
+  @Test
+  void testEmptyActivity() {
```

Comment:
What is this test testing that isn't already covered by the new unit tests? I think we can remove this one.

Reply:
Removed!